lib/statsd/instrument/prometheus/batched_prometheus_sink.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module StatsD
+  module Instrument
+    module Prometheus
+      class BatchedPrometheusSink < ::StatsD::Instrument::BatchedUDPSink
+        # https://coralogix.com/docs/coralogix-endpoints/
+        DEFAULT_MAX_PACKET_SIZE = 1200000 # 1.2 MB
```

Comment:
Took their 2MB max and cut in half. Our implementation is compressed protobufs, so won't be super-accurate anyways

Reply:
[tiny] can we get some underscores in this number definition so it's a bit easier to read :pray:

Reply:
Added in c354a11